### DISCUSSION TITLE
Are there plans for a new release?

### DISCUSSION THREAD

**Discussion by mil-ad** [asmvik/yabai#1672] [General]
I was wondering if there's a timeline for the next official release?

**Comment by asmvik** [comment]
Closing this since a new release was done couple of days ago.

I don't really have a set release schedule. It mostly comes down to what kinds of changes are being worked on (bug fixes, features, things that break API compatiblity). These days most changes are just bug fixes, so releases can be sporadic. I try to do a fair share of fixes before doing a release, as I dislike the idea of pushing many small releases that each contain 1-2 minor fixes. Although the severity of a bug-fix is taken into consideration here. 

If there is a new macOS version and the scripting-addition requires changes I usually do a release right away, as this is important functionality for myself and plenty other people (I imagine anyway).
